### ISSUE TITLE
SBS with socketio fails on test/acc

### ISSUE THREAD

**Issue by baszoetekouw** [SURFscz/SBS#391]
Note, works locally, works on vm/docker, but not with AWS RDS:

```
Traceback (most recent call last):
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/pymysql/connections.py", line 732, in _read_bytes
    data = self._rfile.read(num_bytes)
  File "/usr/lib/python3.9/socket.py", line 704, in readinto
    return self._sock.recv_into(b)
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/eventlet/green/ssl.py", line 253, in recv_into
    return self._base_recv(nbytes, flags, into=True, buffer_=buffer)
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/eventlet/green/ssl.py", line 268, in _base_recv
    read = self.read(nbytes, buffer_)
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/eventlet/green/ssl.py", line 187, in read
    return self._call_trampolining(
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/eventlet/green/ssl.py", line 162, in _call_trampolining
    return func(*a, **kw)
  File "/usr/lib/python3.9/ssl.py", line 1099, in read
    return self._sslobj.read(len, buffer)
ssl.SSLError: [RSA] RSA lib (_ssl.c:2622)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/sqlalchemy/engine/base.py", line 1057, in _rollback_impl
    self.engine.dialect.do_rollback(self.connection)
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/sqlalchemy/engine/default.py", line 682, in do_rollback
    dbapi_connection.rollback()
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/pymysql/connections.py", line 480, in rollback
    self._read_ok_packet()
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/pymysql/connections.py", line 692, in _read_packet
    packet_header = self._read_bytes(4)
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/pymysql/connections.py", line 738, in _read_bytes
    raise err.OperationalError(
pymysql.err.OperationalError: (2013, 'Lost connection to MySQL server during query ([RSA] RSA lib (_ssl.c:2622))')

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/eventlet/wsgi.py", line 569, in handle_one_response
    result = self.application(self.environ, start_response)
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/flask/app.py", line 2548, in __call__
    return self.wsgi_app(environ, start_response)
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/flask_socketio/__init__.py", line 43, in __call__
    return super(_SocketIOMiddleware, self).__call__(environ,
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/engineio/middleware.py", line 74, in __call__
    return self.wsgi_app(environ, start_response)
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/flask/app.py", line 2541, in wsgi_app
    ctx.pop(error)
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/flask/ctx.py", line 415, in pop
    app_ctx.pop(exc)
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/flask/ctx.py", line 253, in pop
    self.app.do_teardown_appcontext(exc)
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/flask/app.py", line 2398, in do_teardown_appcontext
    self.ensure_sync(func)(exc)
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/flask_sqlalchemy/__init__.py", line 899, in shutdown_session
    self.session.remove()
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/sqlalchemy/orm/scoping.py", line 181, in remove
    self.registry().close()
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/sqlalchemy/orm/session.py", line 1809, in close
    self._close_impl(invalidate=False)
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/sqlalchemy/orm/session.py", line 1851, in _close_impl
    transaction.close(invalidate)
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/sqlalchemy/orm/session.py", line 923, in close
    transaction.close()
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/sqlalchemy/engine/base.py", line 2416, in close
    self._do_close()
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/sqlalchemy/engine/base.py", line 2639, in _do_close
    self._close_impl()
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/sqlalchemy/engine/base.py", line 2625, in _close_impl
    self._connection_rollback_impl()
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/sqlalchemy/engine/base.py", line 2617, in _connection_rollback_impl
    self.connection._rollback_impl()
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/sqlalchemy/engine/base.py", line 1059, in _rollback_impl
    self._handle_dbapi_exception(e, None, None, None, None)
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/sqlalchemy/engine/base.py", line 2124, in _handle_dbapi_exception
    util.raise_(
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/sqlalchemy/util/compat.py", line 208, in raise_
    raise exception
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/sqlalchemy/engine/base.py", line 1057, in _rollback_impl
    self.engine.dialect.do_rollback(self.connection)
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/sqlalchemy/engine/default.py", line 682, in do_rollback
    dbapi_connection.rollback()
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/pymysql/connections.py", line 480, in rollback
    self._read_ok_packet()
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/pymysql/connections.py", line 692, in _read_packet
    packet_header = self._read_bytes(4)
  File "/opt/sbs/sbs-env/lib/python3.9/site-packages/pymysql/connections.py", line 738, in _read_bytes
    raise err.OperationalError(
sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (2013, 'Lost connection to MySQL server during query ([RSA] RSA lib (_ssl.c:2622))')
(Background on this error at: https://sqlalche.me/e/14/e3q8)
```

**Comment by baszoetekouw** [comment]
Fixed in d87af26 and 47a3702

**Comment by baszoetekouw** [comment]
more proper fix is https://github.com/SURFscz/SRAM-deploy/pull/368 and https://github.com/SURFscz/SBS/pull/393
